eslint.config.mjs

```diff
@@ -751,4 +783,44 @@ export default typescript.config([
       'no-console': 'off',
     },
   },
+  {
```

Comment:
Order matters for these two new sections. They're below the `files/sentry-test` section.

--- 

We lose a rule in favor of reducing the complexity of the config a little:
- `files/sentry-test` prevents importing `sentry/locale`
- For test files inside gsApp and gsAdmin we're overriding the locale import restriction. 

If we were ok with a longer config then we could add sections called `files/gsAdmin-test` AND also `files/gsApp-test` and add the locale import along with the existing imports that are here... i'm choosing not to.